Packs/Slack/Integrations/SlackV3/SlackV3_test.py

```diff
@@ -188,7 +188,7 @@ def set_integration_context(integration_context):
 
```

Comment:
I only commented about get user, but pleas do a re think of what tests are needed, because with this change of code, we clearly need to add uint tests, for example yto the safe mode flow and for the use if chnnelID in input and for searching by email.